perf_test/blas/blas3/KokkosBlas3_perf_test.cpp

```diff
@@ -238,14 +243,15 @@ int main(int argc, char **argv) {
   options.blas_routines        = std::string(DEFAULT_BLAS_ROUTINES);
   options.blas_args.team_size  = DEFAULT_TEAM_SIZE;
   options.blas_args.vector_len = DEFAULT_VECTOR_LEN;
+  options.blas_args.use_auto   = DEFAULT_USE_AUTO;
 
   options.blas_args.trmm.trmm_args = DEFAULT_TRMM_ARGS;
   options.blas_args.trmm.alpha     = DEFAULT_TRMM_ALPHA;
 
   options.blas_args.gemm.gemm_args = DEFAULT_GEMM_ARGS;
   options.blas_args.gemm.alpha     = DEFAULT_GEMM_ALPHA;
```

Comment:
@e10harvey Can we also have default value for beta and allow users to input beta?

Reply:
Thanks for catching this bug, @vqd8a. Fixing now.